app/src/main/java/kr/ac/konkuk/gdsc/hyeseon/util/context/ContextExt.kt

```diff
@@ -0,0 +1,35 @@
+package kr.ac.konkuk.gdsc.hyeseon.util.context
+
+import android.app.Activity
+import android.content.Context
+import android.view.View
+import android.view.inputmethod.InputMethodManager
+import android.widget.Toast
+import androidx.annotation.ColorRes
+import androidx.annotation.DrawableRes
+import androidx.annotation.StringRes
+import androidx.core.content.ContextCompat
+import com.google.android.material.snackbar.Snackbar
+
+fun Context.hideKeyboard(view: View) {
+    val inputMethodManager = getSystemService(Activity.INPUT_METHOD_SERVICE) as InputMethodManager
+    inputMethodManager.hideSoftInputFromWindow(view.windowToken, 0)
+}
+
+fun Context.toast(message: String) {
+    Toast.makeText(this, message, Toast.LENGTH_SHORT).show()
+}
+
+fun Context.longToast(message: String) {
+    Toast.makeText(this, message, Toast.LENGTH_LONG).show()
+}
+
+fun Context.snackBar(anchorView: View, message: () -> String) {
+    Snackbar.make(anchorView, message(), Snackbar.LENGTH_SHORT).show()
+}
```

Comment:
```suggestion
inline fun Context.snackBar(anchorView: View, message: () -> String) {
    Snackbar.make(anchorView, message(), Snackbar.LENGTH_SHORT).show()
}
```

이런식으로 함수를 인라인화하면 자바 바이트코드 단에서 패러미터로 들어가는 익명함수객체를 만들지 않고 컴파일러가 직접 해당 코드를 넣어주어 조그마한 성능의 이점을 가져갈 수 있습니다.